pod.go

```diff
@@ -144,4 +158,16 @@ func (pc *podCollector) collectPod(ch chan<- prometheus.Metric, p v1.Pod) {
 		addGauge(descPodContainerStatusReady, boolFloat64(cs.Ready), cs.Name)
 		addCounter(descPodContainerStatusRestarts, float64(cs.RestartCount), cs.Name)
 	}
+
+	nodeName := p.Spec.NodeName
+	for _, c := range p.Spec.Containers {
+		req := c.Resources.Requests
+		cpu := req[v1.ResourceCPU]
+		mem := req[v1.ResourceMemory]
+
+		addGauge(descPodContainerRequestedCpuMilliCores, float64(cpu.MilliValue()),
```

Comment:
I'm wondering whether we should not add the metric when `cpu`/`mem` is `nil`, wdyt? Right now this produces metrics with a 0 value when no resource requests are set. Either way we need a test where no resource requests are set.